docs/changelog/127651.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 127651
+summary: Add snapshot attribute to .fleet-agents docs
```

Comment:
```suggestion
summary: Add complete attribute to .fleet-agents docs
```

Reply:
Thanks for catch that a887bc9b47ec158db74447948ab0aa93e923c9e0